app/views/ChooseProvider.js

```diff
@@ -228,103 +229,105 @@ class ChooseProviderScreen extends Component {
+                    }>
+                    <Image source={saveIcon} style={styles.saveIcon} />
+                  </TouchableOpacity>
+                </View>
+                <FlatList
+                  data={this.state.selectedAuthorities}
+                  renderItem={({ item }) => (
+                    <View style={styles.flatlistRowView}>
+                      <Typography style={styles.item} use={'body3'}>
+                        {item.key}
+                      </Typography>
+                      <TouchableOpacity
+                        onPress={() => this.removeAuthorityFromState(item)}>
+                        <Image source={closeIcon} style={styles.closeIcon} />
+                      </TouchableOpacity>
+                    </View>
                   )}
-                  onSubmitEditing={() =>
-                    this.addCustomUrlToState(this.state.urlText)
-                  }
                 />
-                <TouchableOpacity
-                  onPress={() => this.addCustomUrlToState(this.state.urlText)}>
-                  <Image source={saveIcon} style={styles.saveIcon} />
-                </TouchableOpacity>
-              </View>
-            </>
-          ) : (
-            <>
-              <View
-                style={[
-                  styles.flatlistRowView,
-                  { display: this.state.displayUrlEntry },
-                ]}>
-                <DynamicTextInput
-                  onChangeText={text => {
-                    this.setState({
-                      urlText: text,
-                    });
-                  }}
-                  value={this.state.urlText}
-                  autoFocus={this.state.urlEntryInProgress}
-                  style={[styles.item, styles.textInput]}
-                  placeholder={languages.t('label.enter_authority_url')}
-                  onSubmitEditing={() =>
-                    this.addCustomUrlToState(this.state.urlText)
-                  }
-                />
-                <TouchableOpacity
-                  onPress={() => this.addCustomUrlToState(this.state.urlText)}>
-                  <Image source={saveIcon} style={styles.saveIcon} />
-                </TouchableOpacity>
-              </View>
-              <FlatList
-                data={this.state.selectedAuthorities}
-                renderItem={({ item }) => (
-                  <View style={styles.flatlistRowView}>
-                    <Typography style={styles.item} use={'body3'}>
-                      {item.key}
-                    </Typography>
-                    <TouchableOpacity
-                      onPress={() => this.removeAuthorityFromState(item)}>
-                      <Image source={closeIcon} style={styles.closeIcon} />
-                    </TouchableOpacity>
-                  </View>
-                )}
-              />
-            </>
-          )}
-        </View>
-
+              </>
+            )}
+          </View>
+        </ScrollView>
```

Comment:
Now if I add a couple of providers, I'm unable to scroll to see the full list and the "Add" button is off screen:

<img width="320" alt="Screen Shot 2020-04-30 at 8 50 27 AM" src="https://user-images.githubusercontent.com/702990/80731412-bba8a600-8abf-11ea-96c5-7290534dd4a2.png">